### PR TITLE
fix: use status.managedProcessing to get the pipelineRun for Release

### PR DESCRIPTION
**Pull request by sahil143**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-1683

## Description

Release service updated the status of Releases as part of https://issues.redhat.com/browse/RELEASE-1109 `status.managedProcessing` should be used instead of `status.processing` to get the data for PipelineRun

## Type of change

- [ ] Feature
- [X] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Browser conformance: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/Releases/ReleaseOverviewTab.tsx [hunk 1]
```diff
@@ -25,11 +25,16 @@ type ReleaseOverviewTabProps = {
   release: ReleaseKind;
 };
 
+const getPipelineRunFromRelease = (release: ReleaseKind): string => {
+  // backward compatibility until https://issues.redhat.com/browse/RELEASE-1109 is released.
+  return release.status?.processing?.pipelineRun ?? release.status?.managedProcessing?.pipelineRun;
```
Comment (abhinandan13jan) [inline src/components/Releases/ReleaseOverviewTab.tsx:30]: Should be better to use
 release.status?.managedProcessing?.pipelineRun ?? release.status?.processing?.pipelineRun

Comment (sahil143) [reply src/components/Releases/ReleaseOverviewTab.tsx:30]: I don't think it really matters here... it's a nullish coalescing.. if one is undefined/null, the other will be picked up. And we will have one prop at a time.

**Pull request merged**